test/IDE/complete_expr_postfix_begin.swift

```diff
@@ -574,8 +578,7 @@ class InAccessorTestQualified {
     get {
       let _ = self.#^OWN_ACCESSOR_13^#
 // OWN_ACCESSOR_13: Begin completions
-// OWN_ACCESSOR_13-DAG: Decl[InstanceVar]/CurrNominal:      inAccessorProp[#Int#];
-// OWN_ACCESSOR_13: End completions
+// OWN_ACCESSOR_13: Decl[InstanceVar]/CurrNominal:      inAccessorProp[#Int#];
```

Comment:
What's the intent of this change?

Reply:
I was just making it consistent with all the other accessors, it doesn't really matter though since there's only a single match 🤷